### PR TITLE
Resolve: Update to latest xchain-binance

### LINKED ISSUE #1725: Update to latest xchainjs/xchain-binance@5.3.1
Test `BNB` on `testnet`:
- SWAP
- ADD
- WITHDRAW
- SEND 
- Tx history

### PR DESCRIPTION
**Pull request by gromxyz**
Check:
- [x] SWAP
- [x] ADD
- [x] WITHDRAW
- [x] SEND
- [x] Tx history

Closes #1725

### CODE REVIEW

**Review by veado — APPROVED** [review]

**Review by veado — CHANGES_REQUESTED** [review]
`@binance-chain/javascript-sdk` needs to be bumped, too.

- `4.2.0` https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-binance/package.json#L46
- Currently  `4.1.1` `https://github.com/thorchain/asgardex-electron/blob/57b0493c45a429e6a82b468653d251ef1490dded/package.json#L64`

**Review by veado — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     "@thorchain/asgardex-util": "^0.9.1",
     "@thorchain/ledger-thorchain": "^0.1.0-alpha.1",
     "@types/electron-devtools-installer": "^2.2.0",
-    "@xchainjs/xchain-binance": "^5.3.0",
```
Comment (Knowledge231) [inline package.json:78]: Cool

Comment (Knowledge231) [inline package.json:78]: 😁

**Review by Knowledge231 — COMMENTED** [review]
Good

**Pull request merged**